preprocess/analysis/language_analysis.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 23 16:07:17 2020
+
+@author: robert
+"""
+
+# Imports the Google Cloud client library
+from google.cloud import language
+from google.cloud.language import enums
+from google.cloud.language import types
+
+import json
+
+# See readme about how to setup mykey.json
+CLIENT = language.LanguageServiceClient.from_service_account_json("mykey.json")
+ENCODING_TYPE = enums.EncodingType.UTF8
+
+
+def round_to_1(number):
+  """Rounds number to the nearest tenth. 
+
+  Cloud language api will automatically return floats to like 15 decimal 
+  places. It's convenient to just round down. 
+
+  Args: 
+    number: float or an int.
+  
+  Returns:
+    a float to tens precisions.
+  """
+  
+  return float(round(number, 1))
+
+def round_to_2(number): 
```

Comment:
round_to_1 and round_to_2 are duplicated, you can combine both of them to one single function with 2 args instead => something like `def round(number, toPlace)`

Reply:
Also, next time name the function a bit more clear than this, something like: round_to_hundredth and round_to_tenth

Reply:
Done, using the built in method and will just have a comment about sig figs.